metrique-core/src/namestyle.rs

```diff
@@ -51,18 +77,37 @@ impl NameStyle for Identity {
         _snake: &'static str,
         _kebab: &'static str,
     ) -> Cow<'static, str> {
-        Cow::Borrowed(identity)
+        if PREFIX::LEN == 0 {
+            Cow::Borrowed(identity)
+        } else {
+            let mut result = String::with_capacity(PREFIX::LEN + identity.len());
+            PREFIX::extend(&mut result);
```

Comment:
Does this mean we always allocate a new string? I would have this to use the `concat` stuff

Reply:
`fn inflect_name` is deprecated. I should probably just remove it.